core/src/components/table/table-footer/table-footer.tsx

```diff
@@ -26,30 +26,23 @@ export class TdsTableFooter {
   /** Enable pagination and show pagination controls  */
   @Prop({ reflect: true }) pagination: boolean = false;
 
-  /** Sets the default page number. */
-  @Prop({ reflect: true }) defaultPage: number = 1;
+  /** Sets the pagination number. */
+  @Prop({ reflect: true, mutable: true }) paginationValue: number = 1;
 
   /** Sets the number of pages. */
   @Prop({ reflect: true }) pages: number = null;
 
-  /** @deprecated Used to set rows per page, this also enabled automatic pagination of the table. */
-  @Prop({ reflect: true }) rowsPerPage: number;
-
-  /** <b>Client override</b> Used to set the number of columns, use as fallback if the automatic count of columns fail. */
+  /** <b>Client override</b> Used to set the number of columns, use as fallback if the automatic count of columns fails. */
```

Comment:
```suggestion
  /** <b>Client override</b> Used to set the column span of the footer. Use as fallback if the automatic count of columns fails. */
```

Reply:
(I'm adding it as a suggestion because I'm afraid youll get a conflict / force push over it if i commit it directly to your PR without you knowing)